package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "email-alert-frontend",
+  "version": "1.0.0",
```

Comment:
I don't think we want this since we won't maintain it (I believe it's still valid without a version)